packages/pytest-simcore/src/pytest_simcore/db_entries_mocks.py

```diff
@@ -19,52 +20,46 @@
 from simcore_postgres_database.models.projects import ProjectType, projects
 from simcore_postgres_database.models.projects_to_products import projects_to_products
 from simcore_postgres_database.models.services import services_access_rights
-from simcore_postgres_database.models.users import UserRole, UserStatus, users
+from simcore_postgres_database.models.users import UserRole, UserStatus
 from simcore_postgres_database.utils_projects_nodes import (
     ProjectNodeCreate,
     ProjectNodesRepo,
 )
 from sqlalchemy.ext.asyncio import AsyncEngine
 
 from .helpers.postgres_tools import insert_and_get_row_lifespan
+from .helpers.postgres_users import sync_insert_and_get_user_and_secrets_lifespan
 
 
 @pytest.fixture()
 def create_registered_user(
-    postgres_db: sa.engine.Engine, faker: Faker
+    postgres_db: sa.engine.Engine,
 ) -> Iterator[Callable[..., dict]]:
+    """Fixture to create a registered user with secrets in the database."""
```

Comment:
ok so I see you upgraded that one too, which is good. Nevertheless I don't understand why you still keep the other fixtures that do the same as this one. I thought we agreed to use this one everywhere? is there a change?